src/app-layer-parser.c

```diff
@@ -1231,7 +1231,8 @@ static void HandleStreamFrames(Flow *f, StreamSlice stream_slice, const uint8_t
             }
         }
     } else if (flags & STREAM_EOF) {
-        Frame *frame = AppLayerFrameGetById(f, direction, FRAME_STREAM_ID);
+        // if we are not using FRAME_STREAM_TYPE, FRAME_STREAM_ID points to another frame
```

Comment:
this talks about FRAME_STREAM_ID, but that is removed in the PR, so it's not a valid reference

Reply:
right, the commit message seems enough